darts/ad/scorers.py

```diff
@@ -0,0 +1,1582 @@
+        """Calls the function _diff() on every pair (s1,s2) in the list (list_series_1,list_series_2).
+
+        list_series_1 and list_series_2 must have the same length n.
+        Each pair of series in list_series_1 and list_series_2 must be of the same length L and width/dimension W.
+
+        Parameters
+        ----------
+        series_1
+            1st sequence of time series
+        series_2:
+            2nd sequence of time series
+
+        Returns
+        -------
+        TimeSeries
+            Sequence of series of length n
+        """
+        list_series = []
+
+        for (series1, series2) in zip(list_series_1, list_series_2):
+            list_series.append(self._diff(series1, series2))
+
+        return list_series
+
+    def _diff(self, series_1: TimeSeries, series_2: TimeSeries) -> TimeSeries:
+        """Applies the reduced_function to the two time series. Converts two time series into 1.
+
+        series_1 and series_2 must be of the same length L and width/dimension W.
+
+        Parameters
+        ----------
+        series_1
+            1st time series
+        series_2:
+            2nd time series
+
+        Returns
+        -------
+        TimeSeries
+            series of length L and width/dimension W.
+        """
+        series_1, series_2 = _return_intersect(series_1, series_2)
+
+        if self.reduced_function == "abs_diff":
+            return (series_1 - series_2).map(lambda x: np.abs(x))
+        elif self.reduced_function == "diff":
+            return series_1 - series_2
+        else:
+            # found an non-existent reduced_function
+            raise ValueError(
+                f"Metric should be 'diff' or 'abs_diff', found {self.reduced_function}"
+            )
+
+
+# FittableAnomalyScorer
+
+
+class GaussianMixtureScorer(FittableAnomalyScorer):
+    """GaussianMixtureScorer anomaly score
+
+    Wrapped around the GaussianMixture scikit-learn function.
+    Source code: <https://github.com/scikit-learn/scikit-learn/blob/f3f51f9b6/sklearn/mixture/_gaussian_mixture.py#L456>
+    """
+
+    def __init__(
+        self,
+        window: Optional[int] = None,
+        n_components: int = 1,
+        reduced_function=None,
+        component_wise: bool = False,
+    ) -> None:
+
+        """
+        A Gaussian mixture model is trained on the training data when the ``fit()`` method is called.
+        The ``score()`` method will compute the log-likelihood of each sample.
+
+        TODO: stride in training is equal to w, and in score stride is equal to 1. Give the option to change
+        these parameters.
+
+        If 2 time series are given in the ``fit()`` or ``score()`` methods, a reduced function, given as a parameter
+        in the __init__ method (reduced_function), will be applied to transform the 2 time series into 1.
+        Default: "abs_diff"
+
+        component_wise is a boolean parameter in the __init__ method indicating how the model should behave with input
```

Comment:
No need to refer to the `__init__` method. You can simply say it's a parameter of the model (or of the constructor method).